src/styles/components/accordion.css

```diff
@@ -30,7 +31,7 @@
         transform: rotateX(0);
         transition: transform var(--transition-duration) var(--ease);
 
-        .c-accordion_details[open] & {
+        .c-accordion_details.is-active & {
```

Comment:
I think the `[open]` attribute is a better state indicator than than a CSS class.

I can kind of see, from the JS, why you would need a CSS class.

Have you ever seen the source code for [zachleat/details-utils](https://github.com/zachleat/details-utils/blob/v2.0.2/details-utils.js#L98-L212)?